### PR TITLE
Added example and description for stretched clusters.

### PR DESCRIPTION
**Pull request by saloneerege**

### CODE REVIEW

File: examples/stretched_cluster/variables.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+variable "sddc_name"{
+  description = "Name of SDDC."
+  default = "sddc-test"
+}
+
+variable "sddc_region" {
+  description = "The AWS  or VMC specific region."
+  default     = "us-west-2"
```
Comment (rchadalawada) [inline examples/stretched_cluster/variables.tf:23]: does this region support multiple az?

File: examples/stretched_cluster/variables.tf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+variable "vpc_cidr" {
+  description = "SDDC management network CIDR. Only prefix of 16, 20 and 23 are supported."
+  default     = ""
+}
+
+variable "vxlan_subnet" {
```
Comment (rchadalawada) [inline examples/stretched_cluster/variables.tf:31]: Is this going to hold 2 subnets?

File: examples/stretched_cluster/variables.tf (hunk 1 above)
Comment (saloneerege) [reply examples/stretched_cluster/variables.tf:23]: Reached out to VMC team to confirm

File: examples/stretched_cluster/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Example: Deploy a MultiAZ SDDC for stretched cluster
+
+This is an example that demonstrates MultiAZ SDDC management actions like creating, updating and deleting an existing SDDC.
+
+# Stretched Clusters
+
+The Stretched Clusters feature deploys a single SDDC across two AWS availability zones. 
+This option is only available during the SDDC creation. When enabled the default number of ESXi hosts supported in a Stretched Cluster is six. 
+Additional hosts can be added later but must to be done in pairs across AWS availability zones. The Stretched Clusters feature requires an AWS VPC with two subnets, one subnet per availability zone. The subnets determine an ESXi host placement between the two availability zones.
```
Comment (shylasrinivas) [inline examples/stretched_cluster/README.md]: typo - must be done

File: examples/stretched_cluster/variables.tf (hunk 2 above)
Comment (saloneerege) [reply examples/stretched_cluster/variables.tf:31]: customer_subnet_ids is the field used to specify two subnets.

File: examples/stretched_cluster/README.md (hunk 3 above)
Comment (saloneerege) [reply examples/stretched_cluster/README.md]: Done

**Review by shylasrinivas — APPROVED** [review]

**Pull request merged**